script/munin-node-configure

```diff
@@ -372,7 +372,7 @@ sub run_plugin
 
     # Ignore debug output
 	my @response = grep !/^#/, @{ $res->{stdout} };
-    $plugin->log_error('Nothing printed to stdout') unless (scalar @response);
+    $plugin->log_error('Nothing printed to stdout') unless (scalar @response || $mode eq 'suggest');
```

Comment:
I think, you want to omit the error message in case of an empty response, if the `suggest` was requested. Maybe a comment would help to understand your fellow code readers here.
btw.: the spacing in the line above looks different - maybe just fix that, too?